assets/css/table.scss

```
@@ -240,7 +240,7 @@
   .logo {
     margin-right: 0.7em;
     height: 2.5em;
```

Good catch!
From what I remember:
Either height OR width should have absolute values but they can't both have it as that would force icons to be square, even those who aren't supposed to be square.
Obviously the width should be the constraint with the absolute value :sweat_smile: 

```suggestion
    max-height: 2.5em;
```